torch/csrc/api/src/nn/modules/linear.cpp

```diff
@@ -25,6 +25,31 @@ Tensor IdentityImpl::forward(const Tensor& input) {
 
 // ============================================================================
 
+BiasImpl::BiasImpl(BiasOptions options_) : options(options_) {
+  reset();
+}
+
+void BiasImpl::reset() {
+  bias = register_parameter("bias", torch::empty(options.num_features()));
+
+  reset_parameters();
+}
+
+void BiasImpl::reset_parameters() {
+  const auto bound = 1 / std::sqrt(options.num_features());
+  torch::nn::init::uniform_(bias, -bound, bound);
```

Comment:
nit: should initialize from standard normal distribution (`torch::nn::init::normal_`)

Reply:
done. what's the rationale? note that linear initializes its bias from uniform distribution.

what is a sensible stddev for this?

Reply:
Rationale is just that the lab claims it should arbitrarily :p No special reason beyond that really. "The module should accept a constructor argument named num_features that defines the size of its vector parameter, and the vector should be randomly initialized from a standard normal distribution."

> what is a sensible stddev for this?

Standard normal implies 0 mean and 1 stddev

Reply:
oops. my bad!